ipatests/test_integration/test_dns_locations.py

```diff
@@ -287,3 +359,99 @@ def test_change_weight(self):
 
         for ip in (self.replicas[1].ip, self.master.ip):
             self._test_against_server(ip, domain_paris_loc, servers_paris_loc)
+
+    def test_restore_locations_and_weight(self):
+        """Restore locations and weight. Not just for test purposes but also
+        for the following tests"""
+
+        self.master.run_command([
+            'ipa', 'server-mod', self.replicas[0].hostname, '--location='''])
```

Comment:
Did you mean: `"--location=''"` ?

Reply:
Originally yes but I tried two variations like "--location=''" and '--location=""' but always got:

`(Pdb) self.stderr_text
"ipa: ERROR: '': location not found\n"`

or:

`(Pdb) self.stderr_text
'ipa: ERROR: "": location not found\n'
`

''' is actually working for me :)